teos/src/main.rs

```diff
@@ -115,6 +115,20 @@ async fn main() {
         eprintln!("Cannot create network dir: {e:?}");
         std::process::exit(1);
     });
+
+    // Log default data dir
+    log::info!("Default data directory: {:?}", &path);
+
+    // Log config file path
+    log::info!("Config file: {:?}", &conf_file_path);
+
+    // Log datadir path
+    log::info!("Using data directory: {:?}", &path_network);
```

Comment:
Move this up, so the order is:

- Default data directory
- Using data directory
- Config file